Chausie/PageViewController.swift

```diff
@@ -201,7 +201,7 @@ public final class PageViewController: UIViewController {
         }()
 
         coordinator.animate(alongsideTransition: { _ in
-            let contentWidth = self.scrollView.bounds.width
+            let contentWidth = size.width
```

Comment:
Use transition size because `scrollView.bounds`may not be updated in `alongsideTransition `.